base-image/Dockerfile

```diff
@@ -1,8 +1,8 @@
-ARG TestRepo
+ARG TestRepo=custom.repo
```

Comment:
This does seem to fix the issues with recent recent podman versions - I wonder should we just remove the custom.repo default, since it's empty anyway and that may result in a clearer failure if problems happen again?

I think the reason the previous issue wasn't clear is we silently copied custom.repo and ignored the actual TestRepo, so if TestRepo was undefined the COPY would fail and it'd be clearer what the problem was?